src/model/BoundingBox.cpp

```diff
@@ -28,10 +28,12 @@ BoundingBox::~BoundingBox()
 
 void BoundingBox::updateRect(BoundingBoxPoint::TranslationDirection ignoredDirection)
 {
-    QRectF boundingRect = mParentSketchItem->getGraphicsItem()->boundingRect();
-    qreal marginAdjust = 10;
+    QRectF boundingRect = mParentSketchItem->getBounds();
+
+    // Uncomment that if you want a margin on boundingbox
+    //qreal marginAdjust = 10;
     //boundingRect.adjust(-marginAdjust, -marginAdjust, marginAdjust, marginAdjust);
-    setRect(boundingRect);
+    //setRect(boundingRect);
```

Comment:
Yep I've seen it, that made it much easier to understand it while reading your PR!